sdk/lib/math/random.dart

```diff
@@ -10,6 +10,23 @@ part of dart.math;
 /// not suitable for cryptographic purposes.
 ///
 /// Use the [Random.secure]() constructor for cryptographic purposes.
+///
+/// To create a non-negative random integer uniformly distributed in the range
+/// from 0, inclusive, to [max], exclusive, use [nextInt(int max)].
+/// ```dart
+/// var value = Random().nextInt(10); // Value between 0 - 9.
+/// value = Random().nextInt(100) + 50; // Value between 50 - 149
+/// ```
+/// To create a non-negative random floating point value uniformly distributed
+/// in the range from 0.0, inclusive, to 1.0, exclusive, use [nextDouble].
+/// ```dart continued
+/// value = Random().nextDouble(); // Value between 0.0 - 1.0.
```

Comment:
Change this to var value or var doubleValue since this will give error, because the var value type is int in the beginning